daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/EvaluationOrderTest.scala

```diff
@@ -378,12 +378,12 @@ class EvaluationOrderTest extends AnyFreeSpec with Matchers with Inside {
       Value.ValueRecord(
         None,
         ImmArray(
-          None -> Value.ValueParty(alice),
-          None -> Value.ValueParty(bob),
-          None -> Value.ValueParty(alice),
-          None -> Value.ValueTrue,
-          None -> keyValue,
-          None -> emptyNestedValue,
+          None /*"person"  */ -> Value.ValueParty(alice),
+          None /*"obs"     */ -> Value.ValueParty(bob),
+          None /*"ctrl"    */ -> Value.ValueParty(alice),
+          None /*"precond" */ -> Value.ValueTrue,
+          None /*"key"     */ -> keyValue,
+          None /*"nested"  */ -> emptyNestedValue,
```

Comment:
Out of curiosity, since `ValueRecord` takes an `ImmArray[(Option[Name], Value)]`, could we instead have

```suggestion
          Some "person"  -> Value.ValueParty(alice),
          Some "obs"     -> Value.ValueParty(bob),
          Some "ctrl"    -> Value.ValueParty(alice),
          Some "precond" -> Value.ValueTrue,
          Some "key"     -> keyValue,
          Some "nested"  -> emptyNestedValue,
```

?

Reply:
We need to cast the `Name`, they are not String.

Reply:
We need something like `Some(Ref.Name.assertFromString("person"))` which is quite long. 
I can go for that if you think it is better.

Reply:
🤦 I just assumed `Name ~ String`, sorry. 

However I did find some engine tests where we "just" use a string:

https://github.com/digital-asset/daml/blob/b53c7c379964d601986f97f1ebe8ccd6d2694df4/daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/ApiCommandPreprocessorSpec.scala#L118

but there's probably some implicit stuff going on there to allow that, especially since there's no `Some` constructors. Perhaps we could do that here?